### PR TITLE
Add Releases guide to assistant

### PR DESCRIPTION
**Pull request by adhiraj**
Also update the issue page guide.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx [hunk 1]
```diff
@@ -28,9 +28,12 @@ export default class GuideDrawer extends React.Component {
         <div className="assistant-drawer-title">
           {this.props.guide.steps[this.props.step - 1].title}
         </div>
-        <div className="assistant-drawer-message">
-          {this.props.guide.steps[this.props.step - 1].message}
-        </div>
+        <div
+          className="assistant-drawer-message"
+          dangerouslySetInnerHTML={{
```
Comment (adhiraj) [inline src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx]: @lynnagara this is ok for stuff we hardcode right?

Comment (lynnagara) [reply src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx]: Yeah I think so. I assume each message will need to have very different markup / styles, hence the need to define it separately for each one?

Comment (adhiraj) [reply src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx]: Right now I'm not anticipating anything more complicated than links. But yeah each message is custom.

**Comment by adhiraj** [comment]
@Chrissy some tweaks to guide anchor in this diff, in case they collide with something you're doing

**Comment by Chrissy** [comment]
seems likely. thanks for the heads up!

File: src/sentry/assistant/guides.py [hunk 2]
```diff
@@ -6,33 +6,87 @@
...
+                    'events by tags - for example, searching for all events from a specific '
+                    'machine, browser, release etc. The sidebar on the right shows you the '
+                    'distribution of tags for all events in this event group.'),
+                'target': 'tags',
+            },
+            {
+                'title': _('Resolution'),
+                'message': _(
+                    'Resolving an issue removes it from the default dashboard view of unresolved '
+                    'issues. You can ask Sentry to alert you when a resolved issue re-occurs.'),
```
Comment (ehfeng) [inline src/sentry/assistant/guides.py]: You should link them to account settings: https://sentry.io/account/settings/notifications/.

**Comment by adhiraj** [comment]
@ehfeng done

**Review by ehfeng — APPROVED** [review]
The content looks good to me, though the scroll doesn't scroll all the way for me (coming from stacktrace, I didn't actually see the Resolve anchor, because it was only partially in view).

**Comment by adhiraj** [comment]
That will be fixed in Chrissy's diff

**Pull request merged**